lib/view/main_screen/home_screen.dart

```diff
@@ -26,6 +26,8 @@ class HomeScreen extends StatelessWidget {
   HomeScreenController homeScreenController = Get.put(HomeScreenController());
   SingleArticleController singleArticleController =
       Get.put(SingleArticleController());
+  ListArticleController listArticleController =
+      Get.put(ListArticleController());
```

Comment:
I added this line because on this page  there is an error for not finding  listArticleController